test/cmd/README.md

```diff
@@ -0,0 +1,214 @@
+# OpenShift Command-Line Integration Test Suite
+
+This document describes how OpenShift developers should interact with the OpenShift command-line integration test suite.
+
+## Test Structure
+
+The script to run the entire suite lives in [`hack/test-cmd.sh`](./../../hack/test-cmd.sh). All of the test suites that make up
+the parent suite live in `test/cmd`, and are divided by functional area. 
+
+## Running Tests
```

Comment:
i think we should add a little section here on how to setup a test environment; must have openshift installed, should not have a cluster running, etc.